src/tex/ms.tex

```diff
@@ -553,8 +553,9 @@ \subsubsection{Key factors influencing tests of the runaway greenhouse hypothesi
 We will now briefly explore the above drivers.
 
 \textit{Occurrence rate of planets forming steam atmospheres}: The runaway greenhouse climate relies on sufficient amounts of atmospheric water vapor that can act as a greenhouse gas.
-However, already about $\SIrange{\sim 10}{20} \, \mathrm{bar}$ of water vapor -- corresponding to a minor fraction of one Earth ocean and thus the lower limit of water on Earth -- is enough to sustain sufficiently high surface temperatures to keep the planet in a magma ocean stage~\citep{Boukrouche2021,Lichtenberg2021c}.
-The fraction of planets fulfilling these requirements has an impact on the amplitude of the demographic imprint of the runaway greenhouse transition.
+\revii{It was shown that} already about $\SIrange{\sim 10}{20} \, \mathrm{bar}$ of water vapor -- corresponding to a minor fraction of one Earth ocean and thus the lower limit of water on Earth -- is enough to sustain sufficiently high surface temperatures to keep the planet in a magma ocean stage~\citep{Boukrouche2021,Lichtenberg2021c}.
+\revii{However, recent simulations accounting for radiative–convective profiles suggest the existence of cooler runaway greenhouse states that do not necessarily yield a surface magma ocean~\citep{Selsis2023}.
```

Comment:
Suggestion for a few minor additions in the last sentence:
"However, recent simulations accounting for radiative–convective profiles *of near-surface atmospheric layers* suggest the existence of cooler runaway greenhouse states that do not necessarily yield a *molten surface*~\citep{Selsis2023}."

Reply:
Agreed, thank you!